src/features/user/Profile/styles/RedeemModal.module.scss

```diff
@@ -21,30 +21,27 @@
 }
 
 .fixModal {
-  max-height: 279px;
+  max-height: 260px;
```

Comment:
I think the `max-height` can be removed. Keeping this has the effect of reducing the button height when there is an error message below the text field.